From 5cc6d78fd3e0298b997c4fb86c73654cc74844ef Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:51:16 -0400
Subject: [tools] lib/traceevent: Use global QUIET_CC build output

Message-id: <1408953210-26343-189-git-send-email-jolsa@redhat.com>
Patchwork-id: 88958
O-Subject: [PATCH RHEL7.1 BZ1133083 188/322] tools lib traceevent: Use global QUIET_CC build output
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit c3d090f498a4d7a8905c92dbf83ae2aa4810fbef
Author: Jiri Olsa <jolsa@redhat.com>
Date: Thu Dec 19 14:42:02 2013 +0100

description
===========
Using global QUIET_CC build output variable and getting rid of local
print_compile.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/lib/traceevent/Makefile b/tools/lib/traceevent/Makefile
index 022c987..e852a8d 100644
--- a/tools/lib/traceevent/Makefile
+++ b/tools/lib/traceevent/Makefile
@@ -67,6 +67,8 @@ PLUGIN_DIR = -DPLUGIN_DIR="$(DESTDIR)/$(plugin_dir)"
 PLUGIN_DIR_SQ = '$(subst ','\'',$(PLUGIN_DIR))'
 endif
 
+include $(if $(BUILD_SRC),$(BUILD_SRC)/)../../scripts/Makefile.include
+
 # copy a bit from Linux kbuild
 
 ifeq ("$(origin V)", "command line")
@@ -150,7 +152,6 @@ override CFLAGS += $(udis86-flags) -D_GNU_SOURCE
 
 ifeq ($(VERBOSE),1)
   Q =
-  print_compile =
   print_fpic_compile =
   print_shared_lib_compile =
   print_plugin_obj_compile =
@@ -158,7 +159,6 @@ ifeq ($(VERBOSE),1)
   print_install =
 else
   Q = @
-  print_compile =  echo '  CC       '$(OBJ);
   print_fpic_compile =  echo '  CC FPIC  '$(OBJ);
   print_shared_lib_compile = echo '  BUILD    SHARED LIB '$(OBJ);
   print_plugin_obj_compile = echo '  CC FPIC  '$(OBJ);
@@ -188,16 +188,13 @@ do_build_static_lib =    \
  $(RM) $@;  $(AR) rcs $@ $^)
 
 
-define do_compile
- $(print_compile)      \
- $(CC) -c $(CFLAGS) $(EXT) $< -o $(obj)/$@;
-endef
+do_compile = $(QUIET_CC)$(CC) -c $(CFLAGS) $(EXT) $< -o $(obj)/$@;
 
 $(obj)/%.o: $(src)/%.c
- $(Q)$(call do_compile)
+ $(call do_compile)
 
 %.o: $(src)/%.c
- $(Q)$(call do_compile)
+ $(call do_compile)
 
 PEVENT_LIB_OBJS  = event-parse.o
 PEVENT_LIB_OBJS += event-plugin.o
-- 
1.7.1